routes/uploads.js

```diff
@@ -24,13 +27,25 @@ function insertImages (db, scene, userID, callback) {
       url: url,
       status: 'initial',
       user_id: userID,
-      messages: []
+      messages: [],
+      metadata: {
+        acquisition_end: scene.acquisition_end,
+        acquisition_start: scene.acquisition_start,
+        contact: `${scene.contact.name},${scene.contact.email}`,
+        platform: scene.platform,
+        provider: scene.provider,
+        properties: {
+          license: scene.license,
+          sensor: scene.sensor
+        },
+        title: scene.title
+      }
```

Comment:
Potential metadata changes (these all match, but there may be parts that are missing).